home/web/client/inject.js

```diff
@@ -35,11 +38,12 @@ ReactDOM.render(
   // wrapping id used to resest css, see inject.css
   <div className={ stylesReset.reset }>
     <Web3Provider web3={ rawWeb3 }>
-      <TopBar
-        interceptor={ interceptor }
-        web3={ web3 }
-        options={ options }
+      <MuiThemeProvider muiTheme={ muiTheme }>
```

Comment:
This probably doesn't work for injectable dapps. Checking for `document.body` in TopBar rendering was done exactly to solve this issue - maybe we should enclose `ThemeProvider` into `DocumentWaitingThemeProvider` or something.

Also we want topbar logic to kick in as fast as possible (this is why TopBar component was initialized as the first one, but rendering was defered until body is available)